src/java.desktop/share/classes/javax/swing/text/html/CSS.java

```diff
@@ -2862,18 +2861,13 @@ ImageIcon getImage(URL base) {
             lengthMapping.put("cm", 28.3464f);
             lengthMapping.put("pc", 12f);
             lengthMapping.put("in", 72f);
-            int res = 72;
-            try {
-                res = Toolkit.getDefaultToolkit().getScreenResolution();
-            } catch (HeadlessException e) {
-            }
-            // mapping according to the CSS2 spec
-            w3cLengthMapping.put("pt", res / 72f);
-            w3cLengthMapping.put("px", 1f);
-            w3cLengthMapping.put("mm", res / 25.4f);
-            w3cLengthMapping.put("cm", res / 2.54f);
-            w3cLengthMapping.put("pc", res / 6f);
-            w3cLengthMapping.put("in", (float) res);
+            // mapping according to the CSS2.2 spec
```

Comment:
Should we not put the latest CSS link here in addition/instead of in JEditorPane?